src/ResourceManager/Automation/Commands.Automation.Test/ScenarioTests/RunbookTests.cs

```diff
@@ -0,0 +1,97 @@
+        public void TestAutomationStartAndStopRunbook()
+        {
+            RunPowerShellTest("Test-AutomationStartAndStopRunbook -runbookPath ScenarioTests\\Resources\\Test.ps1");
+        }
+
+        [Fact]
+        [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.AcceptanceType, Category.BVT)]
+        public void TestAutomationPublishAndEditRunbook()
+        {
+            RunPowerShellTest("Test-AutomationPublishAndEditRunbook -runbookPath ScenarioTests\\Resources\\Test.ps1 -editRunbookPath ScenarioTests\\Resources\\Test-V2.ps1");
+        }
+
+        [Fact(Skip = "Exception on PUT of jobs and jobSchedules and need to fix test for Playback mode")]
+        [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.AcceptanceType, Category.BVT)]
+        public void TestAutomationConfigureRunbook()
+        {
+            RunPowerShellTest("Test-AutomationConfigureRunbook -runbookPath ScenarioTests\\Resources\\Write-DebugAndVerboseOutput.ps1");
+        }
+
+        [Fact(Skip = "Need to re-record tests with latest version of automation library")]
+        [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.AcceptanceType, Category.BVT)]
+        public void TestAutomationSuspendAndResumeJob()
+        {
+            RunPowerShellTest("Test-AutomationSuspendAndResumeJob -runbookPath ScenarioTests\\Resources\\Use-WorkflowCheckpointSample.ps1");
+        }
+
+        [Fact(Skip = "Exception on PUT of jobs and jobSchedules and need to fix test for Playback mode")]
+        [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.AcceptanceType, Category.BVT)]
+        public void TestAutomationStartRunbookOnASchedule()
+        {
+            RunPowerShellTest("Test-AutomationStartRunbookOnASchedule -runbookPath ScenarioTests\\Resources\\Test.ps1");
+        }
+
+        [Fact(Skip = "Need to re-record tests with latest version of automation library")]
+        [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.AcceptanceType, Category.BVT)]
+        public void TestAutomationStartUnpublishedRunbook()
+        {
+            RunPowerShellTest("Test-AutomationStartUnpublishedRunbook -runbookPath ScenarioTests\\Resources\\Test-WorkFlowWithVariousParameters.ps1");
+        }
+
+        [Fact(Skip = "Exception on PUT of jobs and jobSchedules and need to fix test for Playback mode")]
```

Comment:
Do not add skipped tests - if these tests are necessary, please add them.  If they are not necessary, please remove them altogether.